app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -551,6 +571,7 @@ var schema = yup.object().shape({
   secondary_telephone: yup.string().phone().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
   primary_telephone_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
   secondary_telephone_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
+  international_telephone: yup.string().max(50, 'Max length exceeded, please limit to 200 characters.').nullable(),
```

Comment:
Should say "limit to 50 characters".

Reply:
Also noticed that when `preferred_contact_method` is set, this validation does not work, I think it should be added to the code on 134-200

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1069/commits/c44c099e7284dd1335eed9023d81f655f7a29685